pkg/fake/skus_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// TestSKUExistenceEastUS tests that we are not regressing in our codegen
+func TestSKUExistenceEastUS(t *testing.T) {
```

Comment:
These two functions do the same thing, just with different data. Feels cleaner to just have one function `TestSKUExistence`, which takes in a region (`eastus` or `westcentralus`) and have a map of expectedSKUs (region: list<sku names>) that it can check against.

Reply:
Good shout